schema/artists/trending.js

```diff
@@ -17,9 +15,9 @@ const TrendingArtistsType = new GraphQLObjectType({
   fields: () => ({
     artists: {
       type: new GraphQLList(Artist.type),
-      resolve: results => {
-        const ids = without(keys(results), "cached", "context_type")
-        return Promise.all(ids.map(id => gravity(`/artist/${id}`)))
+      resolve: (deltaResponse, _args, _request, { rootValue: { artistLoader } }) => {
+        const ids = without(keys(deltaResponse), "cached", "context_type")
+        return Promise.all(ids.map(id => artistLoader(id)))
```

Comment:
I wonder instead if we can just do `return artistsLoader({ ids })`, the `/api/v1/artists` endpoint accepts a list of ids. That might be better than fanning out possibly many individual calls.

Reply:
This loader _should_ be batching into a call like you’re describing and thus it takes a single ID so that it will batch from wherever artists either single or multiple are requested. But I’d appreciate another verification that it does indeed work as it should. Can you take a look while I’m gone?